contracts/test/TestRecipient.sol

```diff
@@ -57,7 +58,7 @@ contract TestRecipient is BaseRelayRecipient {
     function dontEmitMessage(string calldata message) public {}
 
     function emitMessageNoParams() public {
-        emit SampleRecipientEmitted("Method with no parameters", _msgSender(), msg.sender, tx.origin, 0, address(this).balance);
+        emit SampleRecipientEmitted("Method with no parameters", _msgSender(), msg.sender, tx.origin, 0, gasleft(), address(this).balance);
```

Comment:
Do you mean rename it or do you mean get `gasleft` before emit? because here it does not make much sense, this method doesn't do anything else, just the emit.

Reply:
not really critical.